config.w32

```diff
@@ -0,0 +1,9 @@
+// $Id$
+// vim:ft=javascript
+
+ARG_ENABLE("stackdriver-trace", "stackdriver_trace support", "no");
+
+if (PHP_STACKDRIVER_TRACE != "no") {
+    EXTENSION('stackdriver_trace', 'stackdriver_trace.c stackdriver_trace_context.c stackdriver_trace_span.c');
```

Comment:
@stanley-cheung I think this is all this extension needs as the code is all in one place (in the root).